### PR TITLE
Malayalam Translation

### PR DESCRIPTION
**Pull request by remedcu**
### Issue Reference
This PR addresses the Issue : Malayalam Translation in the Request List

### Summarize
Alert!: A general testing in mobile view required.

Just another UI Improvement, but checking required for different device view.

### CODE REVIEW

**Comment by vrthra** [comment]
I verified it works on a moto-g.

**Comment by remedcu** [comment]
Another check in the desktop would be appreciated before merging.

**Comment by drtechie** [comment]
L64 to 69 have headings for desktop view

File: mainapp/templates/mainapp/request_list.html [hunk 1]
```diff
@@ -41,12 +41,12 @@
...
 		}
-		#requests-table td:nth-of-type(1):before { content: "District"; }
-    #requests-table td:nth-of-type(2):before { content: "Location"; }
-    #requests-table td:nth-of-type(3):before { content: "Requestee"; }
-    #requests-table td:nth-of-type(4):before { content: "Phone"; }
-    #requests-table td:nth-of-type(5):before { content: "Date"; }
-    #requests-table td:nth-of-type(6):before { content: "More details"; }
+		#requests-table td:nth-of-type(1):before { content: "District - ജില്ല"; }
+    #requests-table td:nth-of-type(2):before { content: "Location - സ്ഥലം"; }
+    #requests-table td:nth-of-type(3):before { content: "Requestee - അപേക്ഷകന്‍റെ പേര്"; }
```
Comment (salmanulfarzy) [inline mainapp/templates/mainapp/request_list.html]: Long strings overlap with adjacent column on mobile devices

image

**Comment by drtechie** [comment]
Tested, LGTM on both mobile and desktop

**Comment by rabeesh** [comment]
Looks fine now

**Pull request merged**